doc/rst/developer/bestPractices/Deprecation.rst

```diff
@@ -11,7 +11,7 @@ Non-public interface features can be removed or modified without deprecation.
 This is the deprecation process:
 
 1. Provide a warning or error message when a deprecated feature is used. This
-   may be in the module code, compiler, or runtime, and can be emitted at
+   maybe in the module code, compiler, or runtime, and can be emitted at
```

Comment:
This is incorrect.  "May" and "be" exist in this sentence on their own - if we wanted to sound more uncertain, "may" in this sentence could be replaced with "might", but the "be" would still remain regardless.  In contrast, "maybe" is more of a state and would be more appropriate if the sentence read "This *is* maybe in the module code ..."